src/main/webapp/app/entities/text-exercise/text-exercise-dialog.component.ts

```diff
@@ -30,6 +31,10 @@ export class TextExerciseDialogComponent implements OnInit {
 
     courses: Course[];
 
+    domainCommandsProblemStatement = [new KatexCommand()];
```

Comment:
We can't reuse the same Command for the 3 editors, they will use the same object and will attach their ace editor to it (so the last editor in line would win, meaning all domain commands would insert text into the third editor).